lib/integrations/awsoidc/deployservice.go

```diff
@@ -43,8 +43,8 @@ var (
 	// requiredCapacityProviders contains the FARGATE type which is required to deploy a Teleport Service.
 	requiredCapacityProviders = []string{launchTypeFargateString}
 
-	// oneAgent is used to define the desired agent count when creating a service.
-	oneAgent = int32(1)
+	// twoAgents is used to define the desired agent count when creating a service.
+	twoAgents = int32(2)
```

Comment:
Do we specify a [placement strategy](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-placement-strategies.html) anywhere to make sure tasks are split between different AZs, or is that the default behavior?

Reply:
It is the default behavior when using Fargate as compute provider:

> Task placement strategies and constraints aren't supported for tasks using the Fargate launch type. Fargate will try its best to spread tasks across accessible Availability Zones. If the capacity provider includes both Fargate and Fargate Spot, the spread behavior is independent for each capacity provider.

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-placement.html#fargate-launch-type

Reply:
Ok cool, can you add this note and a link to a comment here for future readers who may be wondering the same?